web/src/hooks/useRobotAccounts.ts

```diff
@@ -38,34 +38,91 @@ interface createNewRobotAccountForNamespaceParams {
   description: string;
 }
 
-export function useCreateRobotAccount(orgName, {onSuccess, onError}) {
+export function useCreateRobotAccount({namespace, onSuccess, onError}) {
   const queryClient = useQueryClient();
 
-  const createRobotAccntMutator = useMutation(
+  const updateRobotData = async (result) => {
```

Comment:
These calls aren't handled with error handling. A success will still be reported to the user if the robot was created successfully even if these all fail.

Reply:
Yes, I think this was done to make sure a robotAcct is created before adding repos/teams/etc to the robot acct.